<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPage</class>
 <widget class="QWizardPage" name="InputPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;dRonin&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt; basic input signal configuration&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;The controller supports many different types of input signals. Please select the type of input that matches your receiver configuration. If you are unsure, just leave the default option selected and continue the wizard.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;Some input options require the controller to be rebooted before the changes can take place. If an option that requires a reboot is selected, you will be instructed to do so on the next page of this wizard.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QToolButton" name="pwmButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>PWM (One cable per channel)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>PWM</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-pwm-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-pwm-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-pwm-disabled.png</disabledoff>:/setupwizard/resources/bttn-pwm-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ppmButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>PPM (One cable for all channels)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>PPM</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-ppm-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-ppm-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-ppm-disabled.png</disabledoff>:/setupwizard/resources/bttn-ppm-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sbusButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Futaba S.Bus</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>S.Bus</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-sbus-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-sbus-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-sbus-disabled.png</disabledoff>:/setupwizard/resources/bttn-sbus-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sbusNonInvertedButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Futaba S.Bus Non-inverted</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>S.Bus Non-inverted</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-sbusnoninverted-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-sbusnoninverted-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-sbusnoninverted-disabled.png</disabledoff>:/setupwizard/resources/bttn-sbusnoninverted-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="spectrumButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Spektrum satellite (DSM)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>DSM Sat.</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-sat-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-sat-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-sat-disabled.png</disabledoff>:/setupwizard/resources/bttn-sat-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="hottSumDButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Graupner HoTT (SUMD)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>SUMD</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-sumd-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-sumd-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-sumd-disabled.png</disabledoff>:/setupwizard/resources/bttn-sumd-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="hottSumHButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Graupner HoTT (SUMH)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>SUMH</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-sumh-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-sumh-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-sumh-disabled.png</disabledoff>:/setupwizard/resources/bttn-sumh-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="srxlButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Multiplex SRXL</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>SRXL</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-srxl-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-srxl-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-srxl-disabled.png</disabledoff>:/setupwizard/resources/bttn-srxl-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ibusButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>FlySky I-Bus</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>I-Bus</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-ibus-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-ibus-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-ibus-disabled.png</disabledoff>:/setupwizard/resources/bttn-ibus-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbsCrossfireButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Team BlackSheep Crossfire</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border: none }</string>
       </property>
       <property name="text">
        <string>Crossfire</string>
       </property>
       <property name="icon">
        <iconset resource="../wizardResources.qrc">
         <normaloff>:/setupwizard/resources/bttn-tbscrossfire-up.png</normaloff>
         <normalon>:/setupwizard/resources/bttn-tbscrossfire-down.png</normalon>
         <disabledoff>:/setupwizard/resources/bttn-tbscrossfire-disabled.png</disabledoff>:/setupwizard/resources/bttn-tbscrossfire-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../wizardResources.qrc"/>
 </resources>
 <connections/>
</ui>
